x-pack/plugins/uptime/server/lib/synthetics_service/service_api_client.ts

```diff
@@ -94,7 +96,7 @@ export class ServiceAPIClient {
       return axios({
         method,
         url: (this.devUrl ?? url) + '/monitors',
-        data: { monitors: monitorsStreams, output },
+        data: { monitors: monitorsStreams, output, stack_version: this.kibanaVersion },
```

Comment:
Is it important to test this? I started writing tests, but mocking various parts of this class were proving challenging. There aren't currently tests for this file.